packages/headless/src/app/search-app-reducers.ts

```diff
@@ -42,6 +43,7 @@ export const searchAppReducers: ReducersMapObject<SearchAppState> = {
   categoryFacetSearchSet: categoryFacetSearchSetReducer,
   query: queryReducer,
   advancedSearchQueries: advancedSearchQueriesReducer,
+  defaultConstantQueryFilter: defaultConstantQueryFilterReducer,
```

Comment:
I believe this should be part of the `advancedSearchQueries` reducer. It does not make sense to split the logic imo. A register action would set the initial cq value and the `cq` provided no initial cq has been set.